src/targets/gpu/CMakeLists.txt

```diff
@@ -76,7 +76,6 @@ if(NOT MIGRAPHX_USE_COMPOSABLEKERNEL)
         ${CMAKE_CURRENT_SOURCE_DIR}/kernels/include/migraphx/kernels/ck.hpp)
 endif()
 
-include(Embed)
```

Comment:
Does it break if its included again?

Reply:
So if the guard is there, it should be safe to include again?

Reply:
The counter adjustment works as follows:
1. First call to `add_embed_library()` the IDs starts from the initial RESOURCE_ID value
2. Following calls align the RESOURCE_ID at the following multiplicity of 1000

The initial value of RESOURCE_ID is 100, and `migraphx_kernels` IDs start from that value. The `onnx_files` start from 1000, and the `tf_files` start from 2000.

Reply:
We need IDs not to overlap. Otherwise, the resource linker will report duplicate resources with the same ID.

Reply:
Why dont you make the resource id a global property that you increment for each file? So if it starts at 100 and we add_embed_library to 50 files then the next embed library would start increment starting at 150. So you dont need different starting points for different libraries as it will start whever the last library stopped, and there wont be any overlap within the project.

Is the resource IDs a problem if we link in another library like composable kernel that used the same IDs?

Reply:
I changed to a global property, but I want to keep the numbering gaps between libraries for debugging purposes. Resources can be loaded from binary (but no defines showing associations) and have gaps that help identify which ID pool belongs to which library.

Reply:
Reverted the change. Locally, it is building fine (Windows and Linux), but CI started failing to compile. Do not have time right now to investigate that.